src/components/CourseOfferings.js

```diff
@@ -51,20 +52,20 @@ const CourseOffering = ({
         <h3 className="overview__name">{classTypeName}</h3>
         {!!isCamp && <h4 className="overview__weekdays">Monday - Friday</h4>}
         <h4 className="overview__time">{scheduledTimeRange}</h4>
+        <h4 className="overview__dates">{dateRange}</h4>
       </div>
       <ul className="details">
         <li>
-          <strong>Location</strong> {locationName}
+          <strong>Semester</strong> {semester}
         </li>
         <li>
-          <strong>Dates</strong>
-          {` ${
-            sessionCount === 1 ? `1 Session` : `${sessionCount} Sessions`
-          } | ${dateRange}`}
+          <strong>Location</strong> {locationName}
         </li>
         <li>
           <strong>{inSession ? "Next Session" : "First Session"}</strong>{" "}
-          {firstDate}
+          {`${firstDate}  (${
+            sessionCount === 1 ? "1 Session" : `${sessionCount} Sessions`
```

Comment:
all good here!

At one point Maddy was wanting the price to be a per session number (to make it look smaller)

unsure if that's something you want to reopen or not when you talk later this PM

Reply:
I'll ask thanks